src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -41,6 +41,8 @@
 #include <platform/ThreadStackManager.h>
 #endif
 
+#define BLE_ADVERTISEMENT_VERSION 0
```

Comment:
So where this # is coming from ? the specs i guess
Maybe it should be externally configurable .. or all of encoding arrangement for a particular version is done within this file ?

Reply:
Specs did't mention specifically about configuration, so set version in this file.

Reply:
Specs did't mention specifically about configuration, so set version in this file.

Reply:
Specs did't mention specifically about configuration, so set version in this file.